client/src/api/types.ts

```diff
@@ -0,0 +1,12 @@
+import { IUser } from '../models/IUser'
+
+interface IDataResponse {
+    token?: string
+    user?: IUser
+}
+
+export interface IAxiosDataResponse {
```

Comment:
you can create a generic interface for that

Reply:
fix